main.tf

```diff
@@ -62,8 +62,9 @@ resource "pagerduty_escalation_policy" "ep" {
   }
 }
 
-# /* SCHEDULE */
-resource "pagerduty_schedule" "foo" {
+# SCHEDULE
+# see https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/resources/schedule
+resource "pagerduty_schedule" "checkout_service" {
```

Comment:
Interesting. So, is it recommended that we use the naming convention of `checkout_service` for all the objects related to the `checkout_service`? So the schedule would have this name, as well as the escalation policy above?

Reply:
We generally go with one of two names:

* `main` in a module context (also commonly seen as `this`, but that's less clear imo)

* the shortest, most logical reference

In this case, the resources related to `checkout_service` have that as common (= shared) name